pages/app/index.tsx

```diff
@@ -34,6 +34,7 @@ declare const window: ExtendedWindow;
 function isAppLayoutPage(pageId?: string) {
   const appLayoutPages = [
     'app-layout',
+    'app-layout-toolbar',
```

Comment:
This change does nothing, because the check below is `pageId.includes()`, so `app-layout` substring already matches this